tool/dumper/vkgcPipelineDumper.cpp

```diff
@@ -394,6 +394,7 @@ void PipelineDumper::dumpResourceMappingNode(const ResourceMappingNode *userData
   case ResourceMappingNodeType::DescriptorTableVaPtr: {
     char prefixBuf[256];
     for (unsigned i = 0; i < userDataNode->tablePtr.nodeCount; ++i) {
+      snprintf(prefixBuf, 256, "%s.next[%u]", prefix, i);
```

Comment:
I think you might need to assign the result of snprintf to a variable to shut the Windows compiler up, then mark it unused with ((void)myvar) if Sebastian is adding unused variable warnings to vfx. Or maybe prepend (void) to the snprintf line.

Reply:
Aah, I overlooked that, thanks for adding it back.
Is it the Windows compiler that complains about unused results from snprintf? Because neither clang nor gcc complained, so I removed all the (void) conversions and variables in #705.

Reply:
PR705 (https://github.com/GPUOpen-Drivers/llpc/pull/705) was removing results of the snprintf.

so there are several snprintf without return values in the vkgcPipelineDump.cpp. it seem working without Windows compiler complaining.